src/utils.ts

```diff
@@ -25,3 +32,93 @@ export function toAbsoluteUrl(host: string | undefined, path: string): string {
     return host + '/' + path;
   }
 }
+
+/*
+Type with the possible ways to define a color in maplibre
+*/
+export type mbColorDefinition =
+  | PropertyValueSpecification<ColorSpecification>
+  | DataDrivenPropertyValueSpecification<ColorSpecification>;
+
+/*
+A selection of the different maplibre-gl-js properties that refer to a color.
+Taken from maplibre https://github.com/maplibre/maplibre-gl-js/blob/main/src/style-spec/types.g.ts
+*/
+export type layerPaintProperty = {
```

Comment:
it is too bad that we can't get these types from maplibre. 😞

Reply:
I manually picked the properties I wanted to have on our property processing so not sure how we could improve the situation.

I tried just changing the type by `layerPaintProperty = LayerSpecification["paint"]` but I could not make the type work later on the `transformColorProperties` method.

Reply:
It's not worth worrying about. This is fine.